src/infra/core/tests/tests/test_query_service_impl.rs

```diff
@@ -197,28 +197,92 @@ async fn test_dataset_schema_common(catalog: dill::Catalog, tempdir: &TempDir) {
         })
     );
 }
+async fn test_dataset_arrow_schema(catalog: &Catalog, tempdir: &TempDir) {
+    let dataset_alias = create_test_dataset(catalog, tempdir.path()).await;
+    let dataset_ref = DatasetRef::from(dataset_alias);
 
-#[test_group::group(engine, datafusion)]
-#[test_log::test(tokio::test)]
-async fn test_dataset_schema_local_fs() {
+    let query_svc = catalog.get_one::<dyn QueryService>().unwrap();
+    let schema_ref = query_svc.get_schema(&dataset_ref).await.unwrap().unwrap();
+
+    let mut buf = Vec::new();
+    kamu_data_utils::schema::format::write_schema_arrow_json(&mut buf, schema_ref.as_ref())
+        .unwrap();
+
+    let schema_content = String::from_utf8(buf).unwrap();
+    let data_schema_json =
+        serde_json::from_str::<serde_json::Value>(schema_content.as_str()).unwrap();
+
+    assert_eq!(
+        data_schema_json,
+        serde_json::json!({
```

Comment:
Yes! This is very nice example of comparing JSON in tests. Let's do this in other places too.

Reply:
done